vendor/licenses/grammar/sublimetext-nasl.txt

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
Where is this license coming from? I don't see it in the grammar's repository. Did you use the provided script to add the grammar, as per the contribution guidelines?

Reply:
I was working with Tenable's legal department yesterday and submitted a PR for this repo today to clearly add it. The PR also got merged today, see https://github.com/tenable/sublimetext-nasl/blob/master/LICENSE.